devenv/scripts/deploy_e2e_tester.sh

```diff
@@ -0,0 +1,40 @@
+CLUSTER_RESOURCE_GROUP=$(cat state/cluster-info.json | jq '.ClusterResourceGroup' | tr -d '"')
+CLUSTER_NAME=$(cat state/cluster-info.json | jq '.ClusterName' | tr -d '"')
+
+echo "deleting any existing e2e job..."
+az aks command invoke --resource-group $CLUSTER_RESOURCE_GROUP --name $CLUSTER_NAME --command "kubectl delete jobs app-routing-operator-e2e -n kube-system"
```

Comment:
If you add `--ignore-not-found` to this command you can exec it along with the next `kubectl apply` command to avoid a second run command roundtrip (it's kind of slow).